site/_includes/docs-navbar.html

```diff
@@ -36,7 +36,7 @@
         <a class="dropdown-item" href="https://getbootstrap.com/docs/4.0/">v4.0.0</a>
         <div class="dropdown-divider"></div>
         <a class="dropdown-item" href="https://v4-alpha.getbootstrap.com/">v4 Alpha 6</a>
-        <a class="dropdown-item" href="https://getbootstrap.com/docs/3.3/">v3.3.7</a>
+        <a class="dropdown-item" href="https://getbootstrap.com/docs/3.4/">v3.4.0</a>
```

Comment:
Let's keep the original link intact (can change it to v3.3 maybe?) and then add v3.4.

```suggestion
        <a class="dropdown-item" href="https://getbootstrap.com/docs/3.4/">v3.4.0</a>
        <a class="dropdown-item" href="https://getbootstrap.com/docs/3.3/">v3.3.7</a>
```